parcels/field.py

```diff
@@ -404,7 +404,8 @@ def show(self, with_particles=False, animation=False, show_time=0, vmin=None, vm
             import matplotlib.animation as animation_plt
             from matplotlib import rc
         except:
-            raise RuntimeError("Visualisation not possible: matplotlib not found!")
+            print("Visualisation is not possible. Matplotlib not found.")
```

Comment:
I see your point about `try/except`; but that would really make the notebooks less nice to read. Since in principle it's not an error when we can't plot, I think a print statement (ideally warning, see also Issue #158) works better and keeps the notebooks clean

So am merging now, and will then rework as Warnings in #158